gibbon-compiler/src/Gibbon/SExpFrontend.hs

```diff
@@ -20,7 +20,7 @@ import qualified Data.List as L
 import Data.Loc ( Loc(..), Pos(..))
 import qualified Data.Map as M
 import qualified Data.Set as S
-import Data.Text hiding (map, head, init, last, length, zip, reverse, foldr)
+import Data.Text hiding (map, head, init, last, length, zip, reverse, foldr, show)
```

Comment:
this change looks strange. Why?

Reply:
I think this should be resolved now, i made the import of Data.Text qualified.